src/LogicApp/LogicApp.Management.Sdk/changed-function-details-20241220155040.csv

```diff
@@ -0,0 +1,27 @@
+"FileName","CsharpFunction","PowerShellFunction"
```

Comment:
please remove this file

Reply:
Thank you for your help, it has now been removed.